OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic

```diff
@@ -58,7 +58,10 @@ ENV FMW_PKG=fmw_12.2.1.4.0_wls_Disk1_1of1.zip \
 # Copy packages
 # -------------
 COPY $FMW_PKG install.file oraInst.loc /u01/
-RUN chown oracle:oracle -R /u01
+RUN chown oracle:root -R /u01 && \
```

Comment:
This is overkill and a little misleading for the next person with these -R recursive statements.  At this point in the Dockerfile, you have a directory /u01 that is already rw to everyone from line 50.  You copied three files into /u01 for the install process that you will omit later because they are only needed for the install.  Change the COPY command instead, and remove all of these chgrp, chmod statements.

Reply:
Change line 50 from:
    chmod a+xr /u01 && \
TO
    chmod 775 /u01 && \
OR
    chmod rwxrwxrx /u01 && \

Reply:
These changes give me errors installing WebLogic.  u01 is owned by root:root and oracle is owned by oracle:oracle with permissions rwx only for oracle.